.rspec

```diff
@@ -1,2 +0,0 @@
---color
```

Comment:
Yup.  But in this case I like that there isn't a dependency between the `rake spec` and `rspec spec` setups.